src/context/token.tsx

```diff
@@ -51,14 +51,16 @@ export const TokenProvider = ({ children }: React.PropsWithChildren) => {
   const retimer = useRetimer();
   const $setToken = useCallback((input: string | null) => {
     setToken(input);
-
-    retimer(requestIdleCallback(() => {
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- Safari is not supported
+    const requestIdleCallback = window.requestIdleCallback || window.setTimeout;
```

Comment:
Might just import `requestIdleCallback` from `foxact/request-idle-callback`: https://foxact.skk.moe/request-idle-callback

Reply:
I pushed a new commit. The variable for this timerId is because:

<img width="1027" alt="image" src="https://github.com/user-attachments/assets/bcdb722c-c693-4006-a3f8-d9f3f857b019">